src/WeekView/WeekView.js

```diff
@@ -53,26 +57,49 @@ const calculateTimesArray = (
   return times;
 };
 
+const getRawDayOffset = (newDayOffset, options = {}) => {
+  const { left: distanceToLeft = null } = options || {};
+  if (distanceToLeft != null) {
+    // the user wants to see targetDate at <distanceToLeft> from the edge
+    return newDayOffset - distanceToLeft;
+  }
+  return newDayOffset;
+};
+
+const getPageStartDate = (
+  selectedDate,
+  numberOfDays,
+  pageStartAtOptions = {},
+) => {
+  const { left: distanceToLeft = null, weekday = null } =
+    pageStartAtOptions || {};
```

Comment:
since `pageStartAtOptions` has default value there is no need to use `OR` operator
```suggestion
    pageStartAtOptions;
```

Reply:
I can see a lot of places with similar double safety, maybe I miss something?

Reply:
just in case the user provides explicitly `options = null`, to avoid throwing an error